src/main/java/in/tazj/k8s/letsencrypt/acme/CertificateRequestHandler.java

```diff
@@ -39,11 +39,13 @@
   final private String acmeServer;
   final private KeyPairManager keyPairManager;
   final private DnsResponder dnsResponder;
+  final private ImmutableMap<String, String> secretFilenames;
```

Comment:
The field type should be `Map` (the interface), not the particular implementation